src/js/tabs/osd.js

```diff
@@ -1050,6 +1054,11 @@ OSD.chooseFields = function () {
     OSD.constants.STATISTIC_FIELDS = OSD.constants.STATISTIC_FIELDS.concat([
       F.RTC_DATE_TIME
     ]);
+    if (semver.gte(CONFIG.apiVersion, "1.38.0")) {
```

Comment:
@McGiverGim: Sorry for mergeing this, but this should also be `1.39.0`, as per discussion on slack.